ADAL/src/telemetry/ADTelemetryPiiRules.m

```diff
@@ -24,26 +24,30 @@
 #import "ADTelemetryPiiRules.h"
 #import "ADTelemetryEventStrings.h"
 
-static NSDictionary *_piiRules;
+static NSSet *_piiRules;
 
 @implementation ADTelemetryPiiRules
 
 + (void)initialize
 {
-    _piiRules = @{AD_TELEMETRY_KEY_USER_ID: @YES};
+    _piiRules = [[NSSet alloc] initWithArray:@[AD_TELEMETRY_KEY_TENANT_ID,
```

Comment:
nit: since it's now a set, I'd rename it to something like piiFields

Reply:
renamed